### PR TITLE
[9.0][FIX] Greenify test of account_mass_reconcile

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Test adapted to a change of Odoo.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@pedrobaeza

File: account_mass_reconcile/tests/test_scenario_reconcile.py [hunk 1]
```diff
@@ -28,12 +28,19 @@ def setUp(self):
         self.account_fx_expense_id = self.ref("account.income_fx_expense")
         self.acs_model = self.env['account.config.settings']
 
+        self.currency_eur_id = self.ref("base.EUR")
+        company = self.ref('base.main_company')
```
Comment (pedrobaeza) [inline account_mass_reconcile/tests/test_scenario_reconcile.py]: You should call it `company_id` for clarity

Comment (MiquelRForgeFlow) [reply account_mass_reconcile/tests/test_scenario_reconcile.py]: Done

**Review by pedrobaeza — APPROVED** [review]
Fast-tracking for greenifying the branch

**Pull request merged**